content/docs/user-guide/building/requirements.md

```diff
@@ -0,0 +1,127 @@
+# Requirements
+
+When you have a model saved via MLEM and want to use it, the first step is to
+make sure you have the right dependencies and packages. Given a model, MLEM can
+get the list of requirements of different types (including Python, Unix, as well
+as some others).
+
+This complements the [checkenv](/doc/command-reference/checkenv) command.
+
+## Pip based requirements
+
+MLEM can export the installable requirements needed for a model using the
+`mlem build` command, where `model` is the path to model saved via `mlem`
```

Comment:
Link from somewhere in [Python Packages](https://mlem-ai-requirement-bui-vdivzc.herokuapp.com/doc/user-guide/building/pip)?

Reply:
done